tests/e2e/test_pgsql.py

```diff
@@ -0,0 +1,120 @@
+"""
+Module to perform PGSQL workload
+"""
+import logging
+import pytest
+import time
+import re
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating, utils
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, tier1
+from tests import helpers
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def ripsaw(request):
+    # Create Secret and Pool
+    secret = helpers.create_secret(constants.CEPHBLOCKPOOL)
+    pool = helpers.create_ceph_block_pool()
+
+    # Create storage class
+    log.info("Creating a Storage Class")
+    sc = helpers.create_storage_class(sc_name='pgsql-workload',
+                                      interface_type=constants.CEPHBLOCKPOOL,
+                                      secret_name=secret.name,
+                                      interface_name=pool.name)
+    # Create RipSaw Operator
+    ripsaw = RipSaw()
+
+    def teardown():
+        ripsaw.cleanup()
+        sc.delete()
+        secret.delete()
+        pool.delete()
+    request.addfinalizer(teardown)
```

Comment:
this line should be called first so in case something will fail, you'll know the finalizer is already loaded

Reply:
Fixed.